Backends/System/Linux/Sources/Kore/Display.cpp

```diff
@@ -59,9 +59,28 @@ kinc_display_mode_t kinc_display_current_mode(int display) {
 	mode.frequency = 60;
 	Window win = RootWindow(disp, DefaultScreen(disp));
 	XRRScreenResources *res = XRRGetScreenResourcesCurrent(disp, win);
-	XRROutputInfo *out = XRRGetOutputInfo(disp, res, XRRGetOutputPrimary(disp, win));
-	if (out != NULL) {
-		XRRCrtcInfo *crtc = XRRGetCrtcInfo(disp, res, out->crtc);
+	RROutput output = XRRGetOutputPrimary(disp, win);
+	bool skip_frequency_check = false;
```

Comment:
Unused?

Reply:
Oh yeah, was left over. I also fixed the code path when a primary display is actually available :angel:

Reply:
👍 Hehe hoping we can finally detect >60hz displays reliably on Linux.